src/apps/dictionaries/redux/actions.ts

```diff
@@ -391,11 +392,19 @@ export const recursivelyAddConceptsToDictionaryAction = (
     );
 
     const referencesToAdd = await recursivelyFetchToConcepts(
-      sourceUrl,
       concepts.map(concept => concept.id),
-      updateProgress
+      updateProgress,
+      sourceUrl,
+      false
```

Comment:
This is in the wrong order. `sourceUrl` should be the last argument.

Reply:
Also, we should do away with *this* copy of `recursivelyFetchToConcepts()` since we're using that below, right?